go/consensus/tendermint/apps/beacon/beacon.go

```diff
@@ -122,6 +122,7 @@ func (app *beaconApplication) onBeaconEpochChange(ctx *abci.Context, epoch epoch
 	case true:
 		// UNSAFE/DEBUG - Deterministic beacon.
 		entropyCtx = debugEntropyCtx
+		entropy = []byte("luck=1")
```

Comment:
Why is this required? Isn't it the same if you just tweak the identity seeds (with different values ofc)? If it is not, add a comment why this is required.

Reply:
I'll add a comment